server/client_manager.py

```diff
@@ -578,6 +601,9 @@ def get_targets(self, client, key, value, local=False, single=False):
                 elif key == TargetType.IPID:
                     if client.ipid == value:
                         targets.append(client)
+                elif key == TargetType.AFK:
+                     if client in area.afkers:
+                        targets.append(client)
```

Comment:
I assume when the target is `AFK`, then *all* AFK users should be targeted. In this case, add after line 584:
```python
elif key == TargetType.AFK:
    return list(area.afkers)
```

Reply:
Actually that wouldn't work as `area` has not been declared at that line. We are trying to grab all afkers from *each* area as afkers are areawide and not serverwide.

Reply:
```python
elif key == TargetType.AFK:
    return [list(area.afkers) for area in areas]
```

Reply:
Doesn't work either @oldmud0 . It doesn't return the correct thing as it seems. Not sure why

Reply:
I did not accept your original code because it does not show the intent of `TargetType.AFK` in that it entirely disregards `value`. But as you seem so bent on getting this merged, I will allow it.